platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBenchJPDB.java

```diff
@@ -47,10 +44,27 @@ protected VirtualMap<BenchmarkKey, BenchmarkValue> createEmptyMap() {
         final JasperDbBuilder<BenchmarkKey, BenchmarkValue> diskDbBuilder = new JasperDbBuilder<>();
         diskDbBuilder
                 .virtualLeafRecordSerializer(virtualLeafRecordSerializer)
-                .virtualInternalRecordSerializer(new VirtualInternalRecordSerializer())
+                .virtualInternalRecordSerializer(new PathHashRecordSerializer())
                 .keySerializer(new BenchmarkKeySerializer())
                 .storageDir(getTestDir().resolve("jasperdb"))
                 .preferDiskBasedIndexes(false);
         return new VirtualMap<>(LABEL, diskDbBuilder);
     }
+
+    public static void main(final String[] args) throws Exception {
+        final CryptoBenchJPDB bench = new CryptoBenchJPDB();
+        bench.numFiles = 10;
+        bench.numRecords = 100000;
+        bench.maxKey = 1000000;
+        bench.keySize = 8;
+        bench.recordSize = 24;
```

Comment:
The values should rather be set to defaults in `BaseBench` than here.

Reply:
This is a convenience method to run benchmarks locally anyway. So why not keep the properties here to make the experimentations convenient? I don't think it's a bad thing that this kind of configuration is duplicated in `CryptoBenchJPDB` and `CryptoBenchMerkleDb`. After all, they are independent benchmarks.